### DISCUSSION TITLE
CMU 15-799: Special Topics in Database Systems - CS自学指南

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#427] [Announcements]
# CMU 15-799: Special Topics in Database Systems - CS自学指南

CS自学指南

https://csdiy.wiki/%E6%95%B0%E6%8D%AE%E5%BA%93%E7%B3%BB%E7%BB%9F/15799/

**Comment by jiuzhengWang** [comment]
请问有无视频资源……只能校内登陆太可惜了，很想聆听大师的讲解！

**Reply by baichuanzhou** [reply]
点video就可以看了

**Reply by jiuzhengWang** [reply]
点击video发现要用box是吗？这个好像要付费？

**Reply by baichuanzhou** [reply]
如果你说的是15-445的话youtube直接搜有，如果是799的话我就没有办法了

**Comment by Neurnia** [comment]
进了CS61A的链接但是似乎看不了讲座录像？
只有赞助者可以登录看。。。

**Reply by Neurnia** [reply]
大大是看教材配ppt学的吗？课怎么听的啊

**Reply by unuing** [reply]
看videos，不要看recording

**Reply by Neurnia** [reply]
已经解决问题了，谢谢

**Reply by Like-elisi** [reply]
怎么解决的啊，recording能看吗

**Reply by Neurnia** [reply]
recording 不能看，但是你去之前一些季度的可以看到有video的，那是疫情时候的，所以有发到ytb上面的。那些就是可以看的。

**Comment by elleryasimov** [comment]
15-799 是不是就是原来的 15-721

**Reply by Xinyu-Li-123** [reply]
这是两门不同的课。15-799是特定于某个数据库子领域，比如今年（2025 Spring）的就只讲query optimization；15-721是会比较全面地学习数据库内核的各个方面。
